From: Mike Hommey <mh@glandium.org>
Date: Wed, 21 Jun 2017 19:06:40 +0200
Subject: Fix CPU_ARCH test for libjpeg on mips

---
 mozilla/media/libjpeg/moz.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mozilla/media/libjpeg/moz.build b/mozilla/media/libjpeg/moz.build
index 910d89d..aaa41d6 100644
--- a/mozilla/media/libjpeg/moz.build
+++ b/mozilla/media/libjpeg/moz.build
@@ -83,7 +83,7 @@ if CONFIG['LIBJPEG_TURBO_ASFLAGS']:
             'simd/jsimd_arm64.c',
             'simd/jsimd_arm64_neon.S',
         ]
-    elif CONFIG['CPU_ARCH'] == 'mips':
+    elif CONFIG['CPU_ARCH'] == 'mips32':
         SOURCES += [
             'simd/jsimd_mips.c',
             'simd/jsimd_mips_dspr2.S',
